src/sdl2/event.rs

```diff
@@ -1619,13 +1619,6 @@ impl Event {
         }}                      // close unsafe & match
     }
 
-    pub fn unwrap_keymod(keymod_option: Option<keyboard::Mod>) -> keyboard::Mod {
```

Comment:
It's `pub`, so I'm not sure if it can be removed without causing breaking change. It might be not intended as `pub`, however, or might be not visible from outside of crate.

Reply:
It is currently visible as you can see here: https://docs.rs/sdl2/0.32.2/sdl2/event/enum.Event.html#method.unwrap_keymod

However I don't see why it was made public since nothing returns an `Option<Mod>` anymore to my knowledge. Maybe keep it, but annotate it with `#[deprecate]`? See https://github.com/rust-lang/rfcs/blob/master/text/1270-deprecation.md

Also, can you put a note of the deprecation in the changelog? That would be helpful.

Reply:
Restored this method, added deprecation, added an entry to changelog (I didn't mark deprecation with bold font because probably no one is using this API, so it's low importance deprecation).

Tried to build [an app](https://github.com/pacmancoder/rustzx) against modified version, at least keyboard input works, including detection of pressed shift and control keys (not tested all modifiers).

Also added tests for unknown bits.